code/modules/clothing/head/helmet.dm

```diff
@@ -72,3 +73,65 @@
 	cold_protection = HEAD
 	min_cold_protection_temperature = SPACE_HELMET_MIN_COLD_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.5
+
+/obj/item/clothing/head/helmet/lightriot
```

Comment:
Please rename object type in accordance with snake case style (i.e. "light_riot").